src/parser/warrior/fury/modules/talents/Warpaint.tsx

```diff
@@ -29,11 +29,11 @@ class Warpaint extends Analyzer {
 
   onPlayerDamageTaken(event: DamageEvent) {
     if (this.selectedCombatant.hasBuff(SPELLS.ENRAGE.id)) {
-      const preMitigatedDamage = (event.amount + event.absorbed) / (1 - REDUCTION_BONUS);
+      const preMitigatedDamage = (event.amount + (event.absorbed || 0)) / (1 - REDUCTION_BONUS);
```

Comment:
iirc if the damage is completely absorbed then amount will be undefined so you should check for that... 
Might be worth it to just make a constant for ((event.amount || 0) + (event.absorbed || 0)) since you use it twice

Reply:
Done